sopel_waifu/waifu.json

```
@@ -1006,7 +1006,9 @@
         "Shirogane Naoto"
     ],
     "Pokemon": [
-        "Misty"
+        "Misty",
+        "Nurse Joy",
+        "Officer Jenny"
```

Alphabetical, because there's no way to keep track of them otherwise.

Added all the gym leaders I could find. Surely there are plenty more characters who deserve to be in here, though.

```suggestion
        "Acerola",
        "Bea",
        "Bugsy",
        "Candice",
        "Cissy",
        "Clair",
        "Dawn",
        "Elesa",
        "Erika",
        "Fantina",
        "Flannery",
        "Gardenia",
        "Hapu",
        "Hilda",
        "Iris",
        "Janine",
        "Jasmine",
        "Officer Jenny",
        "Nurse Joy",
        "Korrina",
        "Lana",
        "Lenora",
        "Lillie",
        "Liza",
        "Luana",
        "Mallow",
        "Marnie",
        "May",
        "Maylene",
        "Mina",
        "Misty",
        "Nessa",
        "Olympia",
        "Rosa",
        "Roxanne",
        "Roxie",
        "Sabrina",
        "Serena",
        "Skyla",
        "Valerie",
        "Viola",
        "Whitney",
        "Winona",
```